deploy/crds/k8s.cni.cncf.io-networkattachmentdefinitions-crd.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 NVIDIA
```

Comment:
Before deploying net-attach-def from macvlanNetwork CRD it was part of multus, but after that, the operator will fail because it can't find that resource for to handle macvlanNetwork CRD logic:

```
{"level":"error","ts":1602747436.0762315,"logger":"cmd","msg":"Manager exited non-zero","error":"no matches for kind \"NetworkAttachmentDefinition\" in version \"k8s.cni.cncf.io/v1\"","stacktrace":"github.com/go-logr/zapr.(*zapLogger).Error\n\t/usr/src/network-operator/.gopath/pkg/mod/github.com/go-logr/zapr@v0.1.1/zapr.go:128\nmain.main\n\t/usr/src/network-operator/.gopath/src/github.com/Mellanox/network-operator/cmd/manager/main.go:151\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:203"}
```

Reply:
So its a matter of order ?
just deploy it conditionally IMO in both places.

it should not be part of the top level __Operator__ CRDs

Reply:
The operator failed to initialize not when deploy a MacvlanNetwork instance, deploying  NetworkAttachmentDefinition is a pre-request for the operator now.
The operator now work directly with the NetworkAttachmentDefinition, so it can be considered as a component of the operator now, it is also similar to what sriov-network-operator-do

Reply:
OK, after discussion, the Operator now watches net-attach-def CRDs as it is required to created/delete net-attach-def CRs.
thus, the CRD should be defined before the operator is deployed.

Im fine with keeping it here.